components/ShareModal/ShareModal.interface.ts

```diff
@@ -1,10 +1,10 @@
-export type ShareProjectData = {
+export interface ShareProjectData {
```

Comment:
this can be kept as `type` as this is used inside the interface.
```suggestion
export type ShareProjectData = {
```